pkgs/development/python-modules/rxv/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, defusedxml
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "rxv";
+  version = "0.6.0";
+
+  propagatedBuildInputs = [ defusedxml requests ];
+
+  # Looks like the tests require networking
+  doCheck = false;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "aa1d707fb4f6d71581aca9a864fb03e62f001b32c835b72ddba5cfdb5c3a661f";
+  };
```

Comment:
please checkout from github to get tests

Reply:
Done